packages/medusa/src/services/order-edit.ts

```diff
@@ -382,4 +386,39 @@ export default class OrderEditService extends TransactionBaseService {
 
     return orderEdit
   }
+
+  async deleteItemChange(
+    orderEditId: string,
+    itemChangeId: string
+  ): Promise<void> {
+    return await this.atomicPhase_(async (manager) => {
+      const itemChange = await this.orderEditItemChangeService_.retrieve(
+        itemChangeId,
+        { select: ["id", "order_edit_id"] }
+      )
+
+      const orderEdit = await this.retrieve(orderEditId, {
+        select: ["id", "confirmed_at", "canceled_at"],
+      })
```

Comment:
thought: just to be sure: can the status be correctly inferred without all the `*_at` fields? might not a concern in this case?

Reply:
Here it will only tell us if it is confirmed or cancelled otherwise it will be marked as created. But in our case we just want not to not be in one of those 2 states. Wdyt?